Select/racer.go

```diff
@@ -0,0 +1,28 @@
+package racer
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// Compares the response times of a and b, returning the fastest one.
```

Comment:
nit: I'd reword this slightly to say `fetching urls a and b` so at a high level we understand what `a` and `b` are supposed to be from the start.

Reply:
ack